clickhouse/base/string_utils.h

```diff
@@ -23,4 +26,37 @@ inline T FromString(const StringView& s) {
    return result;
 }
 
+inline const char* UUIDFormat() {
+   if constexpr (sizeof(long) == 8) {
+      return "%.8lx-%.4lx-%.4lx-%.4lx-%.12lx";
```

Comment:
Please take a look at ["Format constants for the std::fscanf family of functions"](https://en.cppreference.com/w/cpp/header/cinttypes)

Looks like you need to use `PRIu64`/`SCNu64`.